packages/create-app/templates/default-app/app-config.yaml.hbs

```diff
@@ -72,7 +72,7 @@ scaffolder:
 
 catalog:
   rules:
-    - allow: [Component, System, API, Group, User, Template, Location]
```

Comment:
`Template` is authorized specifically below and serves as an example of per-location `allow` rules. The example below of `Group, User` on `org.yaml` doesn't work if we remove those here, since the location is nested and these rules don't propagate yet.